storm-server/src/test/java/org/apache/storm/blobstore/LocalFsBlobStoreSynchronizerTest.java

```diff
@@ -30,51 +30,46 @@
 import org.apache.storm.shade.org.apache.curator.framework.CuratorFrameworkFactory;
 import org.apache.storm.shade.org.apache.curator.retry.ExponentialBackoffRetry;
 import org.apache.storm.utils.Utils;
-import org.apache.zookeeper.server.NIOServerCnxnFactory;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 /**
  *  Unit tests for most of the testable utility methods
  *  and LocalFsBlobStoreSynchronizer class methods
  */
 public class LocalFsBlobStoreSynchronizerTest {
-  private URI base;
-  private File baseFile;
-  private static Map<String, Object> conf = new HashMap();
-  private NIOServerCnxnFactory factory;
+    private static Map<String, Object> conf = new HashMap<>();
+
+    private File baseFile;
```

Comment:
Should this be annotated with @TEMPDIR - maybe in another Jira

Reply:
ok